subsys/bluetooth/mesh/scheduler_srv.c

```diff
@@ -416,7 +416,7 @@ static void run_scheduler(struct bt_mesh_scheduler_srv *srv)
 static void schedule_action(struct bt_mesh_scheduler_srv *srv,
 			    uint8_t idx)
 {
-	struct tm sched_time;
+	struct tm sched_time = {.tm_sec = 0};
```

Comment:
```suggestion
	struct tm sched_time = {0};
```
would also be permitted by the coding style guidelines, I believe

Reply:
btw compliance checking passed well :)